test/optimizer/histogram_test.cpp

```diff
@@ -147,4 +147,74 @@ TEST_F(HistogramTests, OutputTest) {
   EXPECT_FALSE(os.str().empty());
 }
 
+// NOLINTNEXTLINE
+TEST_F(HistogramTests, MergeTest) {
+  Histogram<int> h1{100};
+  EXPECT_EQ(h1.EstimateItemCount(0), 0);
+  h1.Increment(5);
+  EXPECT_EQ(h1.GetMinValue(), 5);
+  EXPECT_EQ(h1.GetMaxValue(), 5);
+
+  Histogram<int> h2{100};
+  EXPECT_EQ(h2.EstimateItemCount(0), 0);
+  h2.Increment(3);
+  EXPECT_EQ(h2.GetMinValue(), 3);
+  EXPECT_EQ(h2.GetMaxValue(), 3);
+
+  h1.Merge(h2);
+  EXPECT_EQ(h1.GetMinValue(), 3);
+  EXPECT_EQ(h1.GetMaxValue(), 5);
+}
+
+// NOLINTNEXTLINE
+TEST_F(HistogramTests, ClearTest) {
+  Histogram<int> h{100};
+  auto boundaries = h.Uniform();
+  EXPECT_EQ(boundaries.size(), 0);
+
+  EXPECT_EQ(h.EstimateItemCount(0), 0);
+  h.Increment(5);
+  EXPECT_EQ(h.EstimateItemCount(3), 0);
+  EXPECT_EQ(h.EstimateItemCount(4), 0);
+  EXPECT_EQ(h.EstimateItemCount(5), 1);
+  EXPECT_EQ(h.EstimateItemCount(6), 1);
```

Comment:
Mind adding a short comment here why this is one?

Reply:
Done